### ISSUE TITLE
documenting `plot_taxonomy_annotate_ts_alluvial()` and output

### ISSUE THREAD

**Issue by taylorreiter** [Arcadia-Science/sourmashconsumr#36]
```
taxonomy_annotate_df <- read_taxonomy_annotate(Sys.glob("~/github/2022-prjna853785-sourmash/outputs/sourmash_taxonomy/SRR*lineages*csv"))

tmp <- readr::read_csv("https://raw.githubusercontent.com/Arcadia-Science/2022-prjna853785-sourmash/main/inputs/metadata.csv") %>%
  select(query_name = run_accession, time = age_months)

plot_taxonomy_annotate_ts_alluvial(taxonomy_annotate_df, time_df = tmp, tax_glom_level = "genus")
```

![image](https://user-images.githubusercontent.com/23057646/202257981-592829a5-f14d-4715-bd96-9b17dea78a53.png)

**Comment by taylorreiter** [comment]
Change palette:

```
plot_taxonomy_annotate_ts_alluvial(taxonomy_annotate_df, time_df = tmp, tax_glom_level = "genus") +
  ggthemes::scale_fill_tableau(palette = "Tableau 20")
```
![image](https://user-images.githubusercontent.com/23057646/202258653-5dba604f-1e4b-469e-805b-54725e25c142.png)

**Comment by taylorreiter** [comment]
closed by #18
